tests/taskpoller.go

```diff
@@ -119,6 +119,7 @@ func WithExpectedAttemptCount(c int) PollAndProcessWorkflowTaskOptionFunc {
 func WithRetries(c int) PollAndProcessWorkflowTaskOptionFunc {
 	return func(o *PollAndProcessWorkflowTaskOptions) { o.Retries = c }
 }
+func WithoutRetries(o *PollAndProcessWorkflowTaskOptions)           { o.Retries = 1 }
```

Comment:
Hm ... we can't change the taskpoller so that `WithRetries(0)` actually does what you'd expect?

Reply:
`Retries = 1` = means no retries. It should be called `Attempts`. Hopefully I am going to rewrite taskpoller completely.